sdk/storage/storage-blob/MigrationGuide.md

```diff
@@ -0,0 +1,290 @@
+# Guide for migrating to `@azure/storage-blob` v12 from `azure-storage`
+
+This guide is intended to assist in the migration to `@azure/storage-blob` from the legacy `azure-storage` package. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+We assume that you are familiar with `azure-storage`. If you are new to the Azure Storage Blob client library for JavaScript, please refer to the [README](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/storage-blob/README.md) and [samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-blob/samples) rather than this guide.
+
+## Table of contents
+
+- [Migration benefits](#migration-benefits)
+  - [Cross Service SDK improvements](#cross-service-sdk-improvements)
+- [Important changes](#important-changes)
+  - [Package name and structure](#package-name-and-structure)
+  - [Constructing the clients](#constructing-the-clients)
```

Comment:
I would rename this to `Client constructors`. Also, we don't have a section corresponding to this header. We have two sections `Constructing the clients with connection string` and `Constructing the clients with AAD token credentials` instead. I would recommend making these h4 under `Client constructors`

Reply:
Updated